src/v2/Apps/Show/Components/ShowContextCard.tsx

```diff
@@ -34,14 +37,34 @@ const CARD_IMAGE_WIDTHS = [
 export const ShowContextCard: React.FC<Props> = ({ show }) => {
   const { isFairBooth, fair, partner } = show
 
+  const tracking = useTracking()
+  const {
+    contextPageOwnerId,
+    contextPageOwnerSlug,
+    contextPageOwnerType,
+  } = useAnalyticsContext()
+
   const FairInfo = () => {
+    const handleClick = () => {
+      tracking.trackEvent({
+        context_module: ContextModule.presentingFair,
```

Comment:
Are these events not yet in cohesion? there is no action type here!

Reply:
I'm not sure I follow?